src/hooks/useDropdownKeyboardNavigation.tsx

```diff
@@ -68,7 +68,7 @@ const useDropdownKeyboardNavigation = ({
           break;
         case 'ArrowDown':
           if (isInitialNavigation && typeof initialFocusedIndex === 'number') {
-            focusOption('down', initialFocusedIndex);
+            focusOption('down', initialFocusedIndex - 1);
```

Comment:
What is going to happen if initialFocusedIndex is 0?

Reply:
First down key will focus the first item. At least seems to work correctly for times list and language selector

Reply:
Ok, great